apps/support/autoscheduler.inc

```diff
@@ -0,0 +1,68 @@
+AUTOSCHED_SRC=../autoscheduler
+
+WEIGHT_OBJECTS=\
+$(BIN)/weights_head1_conv1_weight.o \
```

Comment:
Doesn't the use of BIN here imply that a separate copy of the autoscheduler is built for each app? Should it be AUTOSCHED_BIN or something?

Reply:
Yep, good point. Let me tweak.

Reply:
PTAL